subsys/net/ip/net_private.h

```diff
@@ -153,30 +153,3 @@ static inline void net_pkt_hexdump(struct net_pkt *pkt, const char *str)
 		buf = buf->frags;
```

Comment:
Regarding the commit "net/private: Remove net_print_frags() function", I would like to keep the net_print_frags() function (we could rename it of course if needed). It is indeed not used anywhere but I have found it useful when debugging things.

Reply:
> I would like to keep the net_print_frags() function (we could rename it of course if needed).

I also wanted to express that concern, but not be accused of complaining too much, decided to concentrate on removed 300 lines of tests ;-)

Reply:
@jukkar ok, though I guess it needs to use the log subsystem and not printk as well.

Reply:
> ok, though I guess it needs to use the log subsystem and not printk as well.

Why? Logging will keep choking unless set to "immediate" mode, and printk() rules anyway.

Reply:
I originally used printk() as the function is suppose to print the output in many pieces that should be in one line. So using logger is no go here.

Reply:
ok